src/ace.F90

```diff
@@ -1089,6 +1097,8 @@ recursive subroutine get_energy_dist(aedist, law, LDIS, IDAT, awr, Q_value)
       allocate(KalbachMann :: aedist)
     elseif (law == 61) then
       allocate(CorrelatedAngleEnergy :: aedist)
+    elseif (law == 66) then
```

Comment:
Eventually the law numbers will be gone and the various laws will be differentiated by the fact that they are different types.